classes/class-wc-connect-shipping-label.php

```diff
@@ -301,27 +301,34 @@ public function meta_box( $post ) {
 				admin_url( 'admin.php' )
 			) );
 
-			$store_options = $this->settings_store->get_store_options();
 			$root_view = 'wc-connect-create-shipping-label';
 			$admin_array = array(
 				'purchaseURL'             => get_rest_url( null, '/wc/v1/connect/label/purchase' ),
 				'addressNormalizationURL' => get_rest_url( null, '/wc/v1/connect/normalize-address' ),
 				'getRatesURL'             => get_rest_url( null, '/wc/v1/connect/shipping-rates' ),
-				'labelStatusURL'          => get_rest_url( null, '/wc/v1/connect/label/' . $order->id . '-%d' ),
+				'labelStatusURL'          => get_rest_url( null, '/wc/v1/connect/label/' . $order->id . '-%d-%d' ),
```

Comment:
It looks like you're following the existing pattern, but why are these endpoint parameters part of the path? Why aren't we using a query string here, or posting some JSON?